### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Order/UpperLowerSetTopology.lean [hunk 1]
```diff
@@ -441,18 +441,18 @@ def map (f : α →o β) : C(WithUpperSetTopology α, WithUpperSetTopology β) w
...
 
 @[simp] lemma toUpperSet_specializes_toUpperSet {a b : α} :
-  toUpperSet a ⤳ toUpperSet b ↔ b ≤ a := by
+    toUpperSet a ⤳ toUpperSet b ↔ b ≤ a := by
   simp_rw [specializes_iff_closure_subset, UpperSetTopology.closure_singleton, Iic_subset_Iic,
     toUpperSet_le_iff]
 
 @[simp] lemma ofUpperSet_le_ofUpperSet {a b : WithUpperSetTopology α} :
-  ofUpperSet a ≤ ofUpperSet b ↔ b ⤳ a := toUpperSet_specializes_toUpperSet.symm
+    ofUpperSet a ≤ ofUpperSet b ↔ b ⤳ a := toUpperSet_specializes_toUpperSet.symm
```
Comment (eric-wieser) [inline Mathlib/Topology/Order/UpperLowerSetTopology.lean:449]: Does the style guide have anything to say here? I'm not sure what the convention is supposed to be.

**Review by eric-wieser — APPROVED** [review]
bors d+

I'd like some clarification about the whitespace changes in the last file, but everything else looks great

File: Mathlib/Topology/Order/UpperLowerSetTopology.lean (hunk 1 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Topology/Order/UpperLowerSetTopology.lean:449]: [Yep](https://leanprover-community.github.io/contribute/style.html#structuring-definitions-and-theorems): "If the theorem statement requires multiple lines, indent the subsequent lines by 4 spaces."

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Pull request closed without merging**